src/views/event-meta.php

```diff
@@ -8,23 +8,24 @@
 }
 
 $events_label_singular = tribe_get_event_label_singular();
-
 ?>
 
 <table id="event-meta" class="eventtable">
 	<colgroup>
+		<col style="width:10%; min-width: 80px;">
 		<col style="width:15%">
-		<col style="width:85%">
+		<col style="width:75%">
 	</colgroup>
 	<tbody>
 	<tr>
-		<td colspan="2" class="tribe_sectionheader">
+		<td colspan="3" class="tribe_sectionheader">
 			<h4><?php echo esc_html( sprintf( __( 'Additional %1$s Fields', 'tribe-events-calendar-pro' ), $events_label_singular ) ); ?></h4>
```

Comment:
Aren' we using our own text domain here?

Reply:
Ah crap on that colspan. :)

No, text domain is good here as the string is the same as in ECP an this way we don't need to translate it again.

Reply:
So, done. :)